util/exercise/Cargo.toml

```diff
@@ -1,12 +1,14 @@
 [package]
 name = "exercise"
-version = "1.0.0"
+version = "1.1.0"
```

Comment:
For future reference, why did you bumped the minor version and not the build version (e.g. `1.0.1`)? 
I thought the minor version is updated when a new  feature is added, which is not a case here (I may be wrong, since proper error-handling could be considered a feature)

Reply:
I consider proper error-handling a feature.